### PR TITLE
admin is the default admin group on macOSX

### PR DESCRIPTION
**Pull request by taylor-benson**
### Description

Fixes https://github.com/chef-cookbooks/maven/issues/97

### Issues Resolved

https://github.com/chef-cookbooks/maven/issues/97

### Check List

[ ] All tests pass. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD>
[ ] New functionality includes testing.
[ ] New functionality has been documented in the README if applicable
[ ] All commits have been signed for the Developer Certificate of Origin. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD>

### CODE REVIEW

File: attributes/default.rb [hunk 1]
```diff
@@ -38,6 +38,8 @@
 default['maven']['group'] = case node['platform_family']
 when 'windows'
   'Administrators'
+when 'mac_os_x'
+  'admin'
```
Comment (tas50) [inline attributes/default.rb]: Shouldn't this be "wheel" on macos?

**Comment by taylor-benson** [comment]
@tas50 I will try wheel when I get home. The main issue I saw was if
setting maven group to admin, I would lose the admin of my users who where
admins. Ill report back soon.

**Comment by taylor-benson** [comment]
@tas50 I have tried "wheel" for the group and I don't loose my privileges. So that seems to work. However I just created a new user as an admin just to test out what group they are in and they only show in the admin group, not the wheel group, first one showed my admin account users, wheel did not

`dscacheutil -q group -a name admin`
`dscacheutil -q group -a name wheel`

**Comment by Xorima** [comment]
Closing due to inactivity. 

If this is still wanted please reopen or open another pull request. Alternatively drop by the #sous-chefs channel on the [Chef Community Slack](http://community-slack.chef.io/) and we'll be happy to help! 

Thanks,
Sous-Chefs

**Pull request closed without merging**